src/components/Forum/ItemDetail.js

```diff
@@ -2,104 +2,93 @@ import React, { useEffect, useState } from "react";
+        });
+      }
+    });
+  }
+
+  return (
+    <div className={styles.mainDiv}>
       <div className={styles.questionSection}>
-  <div className={styles.questionSectionTitle}>{question.title}</div>
-  <div>{question.description}</div>
+        <div style={{ width: "70%" }}>
+          <div className={styles.questionSectionTitle}>{question.title}</div>
+          {question.created && <div style={{ width: '100%', color: "black" }}>Asked  {formatDistanceToNow(question.created, { addSuffix: true })}</div>}        <div className={styles.questionDescription}>{question.description}</div>
+        </div>
+        <div style={{ width: "30%", fontWeight: "bolder", fontSize: "150%" }} className={styles.likes}>LIKES: {question.votedBy.length}</div>
       </div>
-
-
-<div className={styles.commentsContainer}>
-  <div className={styles.commentsTitle}>COMMENTS:</div>
-{question.comments!=undefined && question.comments.length>1 && question.comments.map((comment)=>{
-        
-        return(<div className={styles.commentDiv} style={{display:"flex", flexDirection:"column", width:"100%"}}>
-          <div style={{display:"flex"}}>
-          <div style={{marginRight:"1%"}}>User:{comment.createdBy}</div>
-          {/* <div>Added: {formatDistanceToNow(comment.createdAt)}</div> */}
-          </div>
-          <div>{comment.comment}</div>
-      
+      <div className={styles.commentsContainer}>
+        <div className={styles.commentsTitle}>COMMENTS:</div>
+        {question.comments != undefined && question.comments.length > 1 && question.comments.map((comment) => {
+          return (<div className={styles.commentDiv} style={{ display: "flex", flexDirection: "column", width: "90%" }}>
+            <div style={{ display: "flex", width: "100%" }}>
+              <div style={{ marginRight: "1%", width: "50%", textAlign: "start", color: "black" }}>User:{comment.createdBy}</div>
+              <div style={{ width: "50%", textAlign: "end", color: "black" }}>Added: {comment.createdAt && formatDistanceToNow(comment.createdAt)} ago</div>
+            </div>
+            <div style={{ fontSize: "150%", width: "100%", textAlign: "start" }}>{comment.comment}</div>
           </div>)
-      })}
-</div>
-
-<div className={styles.addComment}>
-<textarea
-        onChange={event => setCommentText(event.target.value)}
-        value={commentText}
-        style={{width:"100%", minHeight:"100px"}}
-      />
-      <div>
-        <button className={styles.addCommentBtn} onClick={handleAddComment}>AddCommnet</button>
+        })}
+      </div>
+      <div className={styles.addComment}>
+        <textarea
+          className={styles.textArea}
+          placeholder="Add comment"
+          onChange={event => setCommentText(event.target.value)}
+          value={commentText}
+        />
+        <div>
+          <button className={styles.addCommentBtn} onClick={handleAddComment}>AddCommnet</button>
+        </div>
       </div>
-</div>
-    
-  </div>
+    </div>
   );
 }
```

Comment:
For now ok, to fix:
- inline styles
- should be splitted in smaller components eg.  addComment, commentList